src/gtk/toga_gtk/widgets/base.py

```diff
@@ -58,8 +61,8 @@ def set_color(self, color):
         pass
 
     def set_background_color(self, color):
-        # By default, background color can't be changed
-        pass
+        if color:
+            self.native.set_background_color(self, color)
```

Comment:
I'm fairly sure this doesn't need the `if color` clause. A widget should always have a background color, even if that color is "Transparent" or "system default". But more importantly, if the method is implemented here, why is is implemented on Button as well?

Reply:
Yes, I guess this should not be here - possibly go back to its original state. Any particular style changes should happen at the individual widget level only.